src/Microsoft.AspNetCore.Localization/Internal/RequestCultureProviderLoggerExtensions.cs

```diff
@@ -22,7 +22,7 @@ static RequestCultureProviderLoggerExtensions()
             _unsupportedUICulture = LoggerMessage.Define<string, IList<StringSegment>>(
                 LogLevel.Warning,
                 2,
-                "{requestCultureProvider} returned the following unsupported cultures '{cultures}'.");
+                "{requestCultureProvider} returned the following unsupported uiCultures '{uiCultures}'.");
```

Comment:
`UI cultures`

Reply:
I updated right now .. it's in the way 😄